src/sqlancer/Main.java

```diff
@@ -172,6 +186,17 @@ private void write(Loggable loggable) {
             }
         }
 
+        public void logReduced(StateToReproduce state) {
+            FileWriter logFileWriter = getReducedWriter();
+            printState(logFileWriter, state);
+            try {
+                logFileWriter.close();
+            } catch (IOException e) {
+                // TODO Auto-generated catch block
```

Comment:
here, we could throw an `AssertionError` instead.

Reply:
Removed.